py/circuitpy_mpconfig.mk

```diff
@@ -198,7 +203,7 @@ endif
 CFLAGS += -DCIRCUITPY_PS2IO=$(CIRCUITPY_PS2IO)
 
 ifndef CIRCUITPY_RANDOM
-CIRCUITPY_RANDOM = $(CIRCUITPY_DEFAULT_BUILD)
+CIRCUITPY_RANDOM = $(CIRCUITPY_ALWAYS_BUILD)
```

Comment:
Doesn't this rely on the OS stub at least?

Reply:
You're right, I missed that.

Reply:
We could think about adding a true default OS stub, though, since it's so simple